src/components/markdown/markdown.tsx

```
@@ -48,25 +49,42 @@ export class Markdown {
     public whitelist?: CustomElementDefinition[] =
         globalConfig.markdownWhitelist;
 
+    /**
+     * Enable lazy loading for images
+     */
+    @Prop()
+    public lazyLoadImages: boolean = false;
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)